config/game_options.txt

```diff
@@ -155,6 +157,7 @@ MIDROUND_ANTAG CLOCKWORK_CULT
 MIDROUND_ANTAG CHANGELING
 MIDROUND_ANTAG WIZARD
 #MIDROUND_ANTAG  MONKEY
+MIDROUND_ANTAG MALF
```

Comment:
Won't this take a random Alice person and turn them into an AI mid round?

Reply:
no, midround_antag is whether or not the round-type will make new antags after the main ones die. You're thinking of `reroll_friendly` which is a var on the gamemode itself.